src/webgl/color/colormap.js

```diff
@@ -1,14 +1,16 @@
 import fs from './colormap.fs.glsl';
 
 function getUniforms(opts = {}) {
-  const {imageColormap} = opts;
+  const {imageColormap, colormapScaler, colormapOffset} = opts;
 
   if (!imageColormap) {
     return;
   }
 
   return {
     u_colormap_texture: imageColormap,
+    colormapScaler: typeof colormapScaler != 'undefined' ? colormapScaler : 0.5,
```

Comment:
Any reason why not to do `colormapScaler || 0.5`? That's easier to read. Are there instances when `colormapScaler` is truthy but not set by the user?

Reply:
`colormapScaler || 0.5` is ok, but `colormapOffset || 0.5` not, I just wanted to use the same pattern for both

Reply:
Right, makes sense. Updated.